<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graph</class>
 <widget class="QDialog" name="Graph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:  rgb(128, 163, 216);</string>
  </property>
  <widget class="QCustomPlot" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>700</width>
     <height>480</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_plot">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>580</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border:  2px solid rgb(161, 199, 242);
    background-color:  rgb(205, 230, 255);
    color:  black;
}

QPushButton:pressed {
	border:  2px solid rgb(161, 199, 242);
    background-color:  rgb(182, 208, 242);
}</string>
   </property>
   <property name="text">
    <string>Plot</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_x_max">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>540</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border:  2px solid rgb(161, 199, 242);
background-color:  rgb(255, 255, 255);
color:  black;
</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_x_max">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>540</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:  2px solid rgb(161, 199, 242);
background-color:  rgb(255, 255, 255);
color:  black;
</string>
   </property>
   <property name="text">
    <string>X_max</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_y_max">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>540</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:  2px solid rgb(161, 199, 242);
background-color:  rgb(255, 255, 255);
color:  black;
</string>
   </property>
   <property name="text">
    <string>Y_max</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_y_max">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>540</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border:  2px solid rgb(161, 199, 242);
background-color:  rgb(255, 255, 255);
color:  black;
</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_x_min">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>500</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:  2px solid rgb(161, 199, 242);
background-color:  rgb(255, 255, 255);
color:  black;
</string>
   </property>
   <property name="text">
    <string>X_min</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_x_min">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>500</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border:  2px solid rgb(161, 199, 242);
background-color:  rgb(255, 255, 255);
color:  black;
</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_y_min">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>500</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border:  2px solid rgb(161, 199, 242);
background-color:  rgb(255, 255, 255);
color:  black;
</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_y_min">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>500</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:  2px solid rgb(161, 199, 242);
background-color:  rgb(255, 255, 255);
color:  black;
</string>
   </property>
   <property name="text">
    <string>Y_min</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
